doc/manual/merging.md

```diff
@@ -256,9 +261,17 @@ final record:
 ## Merging records with metadata
 
 Metadata can be attached to values thanks to the `|` operator. Metadata
-currently includes contract annotations, default value, merge priority, and
+currently include contract annotation, default value, merge priority, and
 documentation. We describe in this section how metadata interacts with merging.
 
+Note that metadata can only be syntactically attached to record fields, at the
```

Comment:
```suggestion
Note that metadata can only be syntactically attached to record fields, with the
```

Reply:
Ah, good catch of this horrible Frenchism :stuck_out_tongue: